crates/solver/src/settlement_access_list.rs

```diff
@@ -0,0 +1,104 @@
+use ethcontract::{Address, H256};
+use reqwest::{
+    header::{HeaderMap, HeaderValue},
+    Client, Url,
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize)]
+pub struct TenderlyRequest {
+    pub network_id: String,
+    pub block_number: u64,
+    pub from: Address,
+    pub input: String,
+    pub to: Address,
+    pub generate_access_list: bool,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct TenderlyResponse {
+    pub generated_access_list: Vec<AccessListItem>,
+}
+
+/// Had to introduce copy of the web3 AccessList because tenderly responds with snake_case fields
+/// and tenderly storage_keys field does not exist if empty (it should be empty Vec instead)
```

Comment:
nit: The wording sounds more like an internal comment than a documentation comment.
I think either making this a normal comment `//` or rewording the comment for users of the type would make sense.

Reply:
True, changed to `//`